sun2000_modbus/registers.py

```diff
@@ -138,17 +138,17 @@ class BatteryEquipmentRegister(Enum):
     GridChargeCutoffSOC = Register(47088, 1, datatypes.DataType.UINT16_BE, 10, "%", AccessType.RW, None)
     # ForcibleChargeDischarge = Register(47100, 1, datatypes.DataType.UINT16_BE, 1, None, AccessType.WO, mappings.ForcibleChargeDischarge) # disabled because not readable (AccessType.WO)
     FixedChargingAndDischargingPeriods = Register(47200, 41, datatypes.DataType.MULTIDATA, None, None, AccessType.RW, None)
-    PowerOfChargeFromGrid = Register(47242, 2, datatypes.DataType.INT32_BE, 100, "kW", AccessType.RW, None)
-    MaximumPowerOfChargeFromGrid = Register(47244, 2, datatypes.DataType.INT32_BE, 100, "kW", AccessType.RW, None)
+    PowerOfChargeFromGrid = Register(47242, 2, datatypes.DataType.INT32_BE, 1, "W", AccessType.RW, None)
+    MaximumPowerOfChargeFromGrid = Register(47244, 2, datatypes.DataType.INT32_BE, 1, "W", AccessType.RW, None)
```

Comment:
Similar to https://github.com/olivergregorius/sun2000_modbus/pull/18/files#r1067455325
Gain must be adjusted to be 0.1, data type and all values must be changed accordingly.